modules/EvseManager/Charger.cpp

```diff
@@ -421,8 +424,11 @@ void Charger::runStateMachine() {
                 break;
             }
 
-            if (new_in_state && lastState != EvseState::PrepareCharging) {
-                signalEvent(types::evse_manager::SessionEventEnum::ChargingResumed);
+            if (new_in_state) {
+                if (last_state not_eq EvseState::PrepareCharging) {
```

Comment:
Not sure if we should mix `!=` and `not_eq`. Better to stay here with `!=`.

Reply:
ok, changed to old C++ style so that it is consistent within the file. On new files I prefer to use and/not etc.